awscli/customizations/scalarparse.py

```diff
@@ -36,13 +38,27 @@ def identity(x):
     return x
 
 
+def choose_default_formatters(timestamp_format):
+    parsers = dict(blob_parser=identity)
+    if timestamp_format == 'none':
+        # For backwards compatibility reasons, we replace botocore's timestamp
+        # parser (which parsers to a datetime.datetime object) with the
+        # identity function which prints the date exactly the same as it comes
+        # across the wire.
+        parsers['timestamp_parser'] = identity
+    elif timestamp_format == 'iso8601':
+        # This is a noop because the underlying botocore behavior already uses
+        # iso8601
```

Comment:
This seems to be contradictory to the above comment, the default botocore parser parses it into a ``datetime.datetime`` object does it not? Assuming this is getting converted using ``str()`` that will get you an ``iso8601`` string but that is only true if every place you pass this in uses ``str()`` to convert the datetime. It's also not exactly ``iso8601`` since they replace the ``T`` separator with a empty space. That is allowed "by mutual agreement" but I typically see iso8601 as rendered with the ``T``.